### PR TITLE
Add bind for /usr/share/libdrm

### LINKED ISSUE #31: /usr/share/libdrm/amdgpu.ids: No such file or directory
When launching on Ubuntu 21.04 with AMD Gpu

### PR DESCRIPTION
**Pull request by AlanGriffiths**
Add bind for /usr/share/libdrm.

Fixes: #31 (At least the initial issue)

Also some drive-by tidy up.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Just a quick question, what did the speed dispatcher do?

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -24,16 +24,17 @@ compression: lzo
 
 apps:
   scummvm:
-    command-chain: ["snap/command-chain/alsa-launch"]
+    command-chain:
+      - snap/command-chain/alsa-launch
+      - snap/command-chain/desktop-launch
```
Comment (lucyllewy) [inline snap/snapcraft.yaml]: This (`desktop-launch`) is added by the extension, so by including it here we're running it twice

Comment (AlanGriffiths) [reply snap/snapcraft.yaml:29]: On my systems nothing observable. But I shouldn't be dropping it in _this PR_.

Will restore.

**Comment by AlanGriffiths** [comment]
> Just a quick question, what did the speed dispatcher do?

On my systems nothing observable. But I shoudn't be dropping it in _this PR_.

Will restore

**Comment by lotharsm** [comment]
Thanks for the patch, @AlanGriffiths!

speech-dispatcher is required for the TTS functionality to work, so it should stay :-)

**Review by lucyllewy — APPROVED** [review]
LGTM

**Comment by AlanGriffiths** [comment]
I think all questions are answered/fixed.

What's the protocol for landing this? I appear to have the rights to merge, but should I wait for another ack?

**Pull request merged**